governance/reviews/cloudevents-08-29-2023.md

```diff
@@ -0,0 +1,210 @@
+# CloudEvents Governance Review
+
+What follows is a governance review and assessment for the _CloudEvents project. This review is carried out by members of the Governance Working Group of TAG Contributor Strategy. The review may have been done because of a change in maturity level for the project, at the request of the TOC, or as a request by the project itself. If requested by the project, the review will be provided to the project maintainers. Otherwise, the review will be submitted to the TOC for their follow-up.
+
+Projects may ask TAG Contributor Strategy for assistance in resolving any issues uncovered by the review. The TAG is available via our [slack channel](https://cloud-native.slack.com/archives/CT6CWS1JN), [email](https://lists.cncf.io/g/cncf-tag-contributor-strategy), [GitHub](https://github.com/cncf/tag-contributor-strategy), or by joining our weekly meetings (listed on the [CNCF public calendar](https://www.cncf.io/calendar/)).
+
+## Summary and Assessment
+
+Status: Needs Work
+
+CloudEvents is mostly executing well on fair and effective governance of the CloudEvents specification and its releases.  Governance is nonpartisan, sophisticated, and appropriate to the publication of a spec used by multiple other cloud native projects.
+
+However, governance documentation and transparency is poor.  While most items that should be in governance docs are present, they are in neither ordered nor organized into the files that one would expect them to be in.  There are also multiple parts to their governance, including subprojects, with no clear high-level explanation of how these parts relate to each other. The subprojects are in a poor state of leadership.
+
+None of the individual issues with clarity, organization, or transparency of governance would be by itself a blocker to graduation.  However, taken together, TAG-CS regards the problems as prohibitive.  The project should work to address at least some of the high priority problems with project governance before being voted on for Graduation.
+
+### Executing the Assessment
+
+This assessment was [requested by contributor Doug Davis](https://github.com/cncf/tag-contributor-strategy/issues/478) as part of CloudEvents' application to Graduate.  
+
+Josh Berkus performed the first review on August 28, 2023.
+
+### Must-Fix Items
+
+The project should fix some (but does not need to fix all) of the issues below before Graduation:
+
+* Disorganization of governance documents
+* Clarity of governance documents
+* Need for complete governance summary
+* Implement full Admin lifecycle
+* Return SDKs to maintained status
+
+### Points of Excellence
+
+The following aspects of governance are exemplary, and can be referenced as examples for other projects to copy:
+
+* The [List of Implementations](https://github.com/cloudevents/spec/blob/main/docs/open-source.md) is a very useful resource, particularly for Specification projects
```

Comment:
Can we use commit hashes in the links to the files in GitHub? The reviews are done using snapshots in the time.